src/journal/journald-server.c

```diff
@@ -348,8 +349,10 @@ static int server_system_journal_open(
                  *
                  * If in persistent mode: create /var/log/journal and the machine path */
 
-                if (s->storage == STORAGE_PERSISTENT)
+                if (s->storage == STORAGE_PERSISTENT) {
                         (void) mkdir_parents(s->system_storage.path, 0755);
+                        (void) chattr_path("/var/log/journal", FS_NOCOW_FL, FS_NOCOW_FL, NULL);
```

Comment:
```suggestion
                        (void) chattr_path("/var/log/journal", FS_NOCOW_FL, FS_NOCOW_FL, NULL);
                        (void) chattr_path(s->system_storage.path, FS_NOCOW_FL, FS_NOCOW_FL, NULL);
```
Since chattr_path is not applying recursively as far as I can tell?

Reply:
The `mkdir_parents()` call only creates /var/log/journal. There's no need for recursive operation since we only want to set the attribute if we create /var/log/journal.